docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -521,6 +531,166 @@ To manage the media controller playlist in your application, you must handle req
+    - SUBTITLES - Ability to receive requests for subtitles mode change from media controller clients.
+    - MODE_360 - Ability to receive requests for spherical (360&deg;) mode change from media controller clients.
+
+2. Media controller complex abilities - each ability consists of several types.
+
+    - playback - Abilities of server's playback actions.
+    - displayRotation - Abilities of setting display orientations.
+    - displayMode - Abilities of setting display modes.
+
+To set abilities on server side you can use `saveAbilities()` method.
+
+```
+server.updatePlaybackState("PLAY");
+server.abilities.playback.next = "YES";
+server.abilities.playback.prev = "YES";
+server.abilities.playback.rewind = "NO";
+server.abilities.playback.forward = "NO";
+
+server.abilities.playback.saveAbilities();
+```
+
+Keep in mind that using `saveAbilities()` is required to save changes of playback abilities into database, otherwise changes will have no effect on the device and clients will not be notified about an update.
+
+To get abilities of the server on the client side:
+
+```
+info = client.getLatestServerInfo();
+console.log("After save:");
+console.log("ability NEXT: " + info.abilities.playback.next);
+console.log("ability PREV: " + info.abilities.playback.prev);
+console.log("ability REWIND: " + info.abilities.playback.rewind);
+console.log("ability FORWARD: " + info.abilities.playback.forward);
+```
+
+You can also monitor server's changes of its abilities by using ` addAbilityChangeListener` method on client side.
+
+```
+/* Client-side code */
+var client = tizen.mediacontroller.getClient();
+
+var listener =
+{
+  onplaybackabilitychanged: function(server, ability)
+  {
+    console.log("playback ability changed, server name: " + server.name + ", abilities: ");
+    console.log(JSON.stringify(ability));
+  },
+  ondisplayrotationabilitychanged: function(server, ability)
+  {
+    console.log("display rotation ability changed, server name: " + server.name + ", ability: ");
+    console.log(JSON.stringify(ability));
+  },
+  ondisplaymodeabilitychanged: function(server, ability)
+  {
+    console.log("displayMode ability changed, server name: " + server.name + ", abilities: ");
+    console.log(JSON.stringify(ability));
+  },
+  onsimpleabilitychanged: function(server, type, ability)
+  {
+    console.log(type + " ability changed, server name: " + server.name + ", ability: " + ability);
+  }
+};
+
+var watchId = client.addAbilityChangeListener(listener);
+
+/* Server-side code */
+var server = tizen.mediacontroller.createServer();
+server.abilities.playback.play = "YES";
+server.abilities.playback.saveAbilities();
+server.abilities.shuffle = "NO";
+server.abilities.repeat = "YES";
+```
+
+You will receive information about ability changes of every active media controller server. To receive information only from selected servers, calling function `subscribe()` is required.
```

Comment:
You will receive -> Implicitly you will receive

Reply:
This is already good.